### DISCUSSION TITLE
Is there any possibility to make video with less space usage?

### DISCUSSION THREAD

**Discussion by aalorart** [akiver/cs-demo-manager#928] [Q&A] [answered]
is there any possibility to make video with less space usage? 10gb for 60s is crazy

**Comment by akiver** [comment, marked as answer]
The disk space is **temporarily** required - images generated by the game (the things that take spaces) are deleted at the end.
Using less space is not possible because we need the images generated by the game to create the video.

**Reply by akiver** [reply]
Update: since [v3.12.0](https://cs-demo-manager.com/changelog#v3120), it's possible to use HLAE to avoid images generation (Windows only).
